modules/dns/README.md

```diff
@@ -31,6 +31,46 @@ The returned `flags` parameter means that when you run a `terraform plan` comman
             # (12 unchanged attributes hidden)
         }
 
+The DNS records can be imported from a file in IBM Cloud Internet Services. This module allows the import of DNS records in two ways:
+
+* using the local file path (if accessible)
+* using the base64 encoded string for the file
+
+To convert the records text file to base64 encoded string:
+```sh
+cat dns_records.txt | base64
+```
+
+When the dns_records are successfully imported then the `local_file` tries to replace the file.
```

Comment:
can we specify via which way? or it happens in both ways?

Reply:
Done